Tasks/AzurePowerShell/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 23
+        "Patch": 24
```

Comment:
I'm still working on testing this change. For username/password auth, both Azure and AzureRM modules should be imported since we don't have a way to control which one the user really wants.